modules/eks-iam-user/variables.tf

```diff
@@ -0,0 +1,37 @@
+variable "name" {
```

Comment:
cant we have only one "name" variable and generate both iam group name and namespace using this name if needed, so that no need to pass also namespace name via "namespace" variable ?

you can remove also aws_account_id variable and use terraform special data/resource to know the value

Reply:
and also the name of module seems not relevant to that it do, it is actually not creating user, it creating iam group and eks namespace and limiting users access into that namespace, can we change the name?